tomcat/src/test/java/org/apache/coyote/http11/HttpPathTest.java

```diff
@@ -0,0 +1,37 @@
+package org.apache.coyote.http11;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("HttpPath 클래스는")
+public class HttpPathTest {
+
+    @DisplayName("HttpPath 객체를 생성할 수 있다.")
+    @Test
+    void create_HttpPath() {
+        // given
+        final String httpPath = "/users";
+
+        // when
+        final var result = HttpPath.from(httpPath);
+
+        // then
+        Assertions.assertThat(result.getPath()).isEqualTo("/users");
```

Comment:
```suggestion
        assertThat(result.getPath()).isEqualTo("/users");
```
Assertions을 static import 한 후 위와 같이 구현하는 것은 어떨까?